### ISSUE TITLE
"Operation will always return the left side of the expression" reported on Process.send_after/3

### ISSUE THREAD

**Issue by levine1726** [rrrene/credo#938]
### Precheck

* Proposals for new features should be submitted via: https://github.com/rrrene/credo-proposals
* For bugs, please do a quick search and make sure the bug has not yet been reported here

### Environment

* Credo version (`mix credo -v`): 1.6.1
* Erlang/Elixir version (`elixir -v`): Elixir 1.12.3 (compiled with Erlang/OTP 24)
* Operating system: Docker container leverage elixir:1.12.3 as its base image

### What were you trying to do?
mix credo on a file containing Process.send_after/3

### Expected outcome
No warnings.

### Actual outcome
I received this warning:
```
[W] Category: warning
┃    ↗  Priority: high
┃
┃       Operation will always return the left side of the expression.
┃       [file stacktrace removed]
┃
┃    __ CODE IN QUESTION
┃
┃    23
┃    24   defp schedule() do
┃    25     Process.send_after(self(), :update, 60_000 * 1)
┃    26   end
┃    27
┃
```

I would think that we would want to exclude this type of warning when applied to send_after/3, since handling :update will produce side effects in our system. Please let me know if my thinking is incorrect here, and thank you for reading!

**Comment by johnf9896** [comment]
Isn't Credo actually complaining about `60_000 * 1`? That operation does always return the left side of the expression.

**Comment by levine1726** [comment]
Well this is embarrassing. Thank you for pointing that out!
